### PR TITLE
chore(build): start building against node 10, 11

### PR DESCRIPTION
**Pull request by ofrobots**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -48,10 +48,18 @@ jobs:
     docker:
       - image: node:9
     <<: *unit_tests
+  node10:
```
Comment (kjin) [inline .circleci/config.yml]: Should we just remove `node4 and `node9`?

Comment (ofrobots) [reply .circleci/config.yml]: That would make this semver-major, and I didn't feel it is costing us anything to keep circle ci building those =D.

Comment (kjin) [reply .circleci/config.yml]: SGTM

**Comment by ofrobots** [comment]
I have no idea why circle is not building this.

**Comment by kjin** [comment]
@ofrobots You need to add `node10`, `node11` [here](https://github.com/google/eventid-js/pull/19/files#diff-1d37e48f9ceff6d8030570cd36286a61R17).

**Comment by ofrobots** [comment]
Thanks for catching that bug. That is unrelated to why circle is not building (it is still not building after fixed that).

**Comment by ofrobots** [comment]
It finally built!

**Pull request merged**